### ISSUE TITLE
Serving static files that are bundled into a nexe binary via express

### ISSUE THREAD

**Issue by chrstew** [nexe/nexe#485]
Hey there, we use express to serve some static files to be accessed over http.
Using the latest nexe beta release I include the files to be bundles via the -r parameter but I cannot access them once running the compiled binary.

Example:
```
var express = require('express');
var app = express();
app.use('/public', express.static('./public'));

app.listen(EXPRESS_PORT, function() {
	console.log( 'Web interface listening on: http://localhost:'+ EXPRESS_PORT );
});
```
I tried a few different variations of the path like `public, .//public, public/, etc`

This file when compiled with:

`nexe -i index.js -o index -r  "./public/**" -t static-9.1.0`

Will result in 404's accessing any `/public` url but works fine as a native node script.

In short: Is there a way to serve static content via nexe, with the contents being bundled in the binary?

**Comment by chrstew** [comment]
Is my question ignored for any specific reason? Sorry for the trouble

**Comment by calebboyd** [comment]
I'm trying to find out why. Forgot to add a label
